src/aggregate_utils.py

```diff
@@ -1,17 +1,39 @@
-from src.download_utils import get_default_params, download_store_data
+from download_utils import get_default_params, download_store_data
```

Comment:
I don't think this change is correct if Python is run from the root of the repository.

Reply:
Sorry I was testing within repo, my bad! I shouldn't have rushed the merge req